### ISSUE TITLE
[Functional] Applications support: Microsoft Office

### ISSUE THREAD

**Issue by Drugoy** [cshnik/TaskSpace#9]
When I tried to add Microsoft Office's Word window to the TaskSpace frame it returned me an error:

```
---------------------------
TaskSpace x64 UNREGISTERED VERSION
---------------------------
Приложение TaskSpace не поддерживает работу с данным приложением.
---------------------------
ОК   
---------------------------
```

Will you ever bring support of that app?
Portable office (thinstall) Word 2010 v14.0.4514.1007 (32-bit) under w7sp1x64.

**Comment by cshnik** [comment]
That is the most sore point about TaskSpace functionality.
Proper behaviour of any application inside TaskSpace depends on its developers: either they used any tricky casts(related to GetParent() or same API calls) inside their apps or not. It is already known that many of Ribbon applications works bad inside TaskSpace, and the most likely reason is that Ribbon Bar's windows are organized complexly compared to regular dialog/frame applications.
Unfortunately, MS Office tools were written in much different ways and some of them were added to restricted list not to spoil impression about TaskSpace when these applications will show UI defects(which are treated as TaskSpace's bugs) or they even will not work at all.
Someone already noticed that WordPad crashes after a few minutes, but TaskSpace continues work. So it looks like we need to restrict WordPad for now since we has no information about how to fix these issues.
This area is very hard to explore, we need to defer this issue.

**Comment by Drugoy** [comment]
Oh. It's just the thing is that I remember that at work I've tried to put Word and Excel windows into TaskSpace frame and it worked frame.
But that happened about a few weeks ago, so I used an earlier version of TaskSpace.
And I don't remember which version of Microsoft Office I used there.

To be honest - I hate any limitations. In my opinion it's better to ask the user something like:
"The window you are trying to put into TaskSpace frame was found in the list of unsupported applications, part of it's functionality may be not working inside the frame. We recommend to use this application in it's own window. This may cause UI defects, partial loss of the functionality and application crash! Are you sure you'd like to proceed with the operation?"
and let the user have the choice. Maybe the lacking functionality is not critical and the user would like to go on using this window in a frame? Who knows.
I didn't experience crashes with Word, but yeah, the buttons in the title bar were unreachable.
